### ISSUE TITLE
Wiki?

### ISSUE THREAD

**Issue by ruehsn** [Disassembler0/Win10-Initial-Setup-Script#225]
Would you consider opening a public Wiki under this project to help document the effect of each option has in Windows?  Most of the function names are pretty self-explanatory but I think Wiki with some screenshots could help remove any doubt.

Thanks for creating this!

**Comment by ruehsn** [comment]
After further consideration, I think a Spreadsheet might be a better format for storing this information.

I've created on at:
https://docs.google.com/spreadsheets/d/1KQ-q7IsyeOLJfF9qX3-0EK33vlqY9LTd5l57OeiPp6E/edit?usp=sharing

**Comment by Disassembler0** [comment]
The tweak library is getting bigger and bigger, so this is probably a good idea.

I would maybe combine both forms. Sometimes it's much easier just to paste an image than to describe what does the tweak do. Other times, the explanation might be lengthy - e.g. the difference between *autorun* and *autoplay* or what the hell is an *NCSI Probe*. So we could have some structured wiki with mixture of paragraphs and images which would cover the explanation, including where to find the option in GUI or GPO, and reasons for keeping/changing (this is actually how the whole script started on my blog) and then we would have a summary table with defaults and possible options. (Also, "average users" shouldn't be using this script in the first place. :) There's a bunch of other tools much more suitable for them.)

**Comment by ruehsn** [comment]
I agree "Average Users" won't be running this script... this column is for if you are going to run the script for non-tech savvy friends/family/co-workers; aka you probably want to the privacy tweaks but show hidden files might confuse them so you would leave that off

**Comment by stepmond** [comment]
je pense que meme les tech ont besoin d'un tutoriel surtout avec les nouvelles fonctionnalités de windows 10, ou concernant des fonctionnalités mals connues comme le llmnr c'est pourquoi je maintiens ce wiki en français [https://www.pcsoleil.fr/optimisation-de-windows-10-grace-a-win10-initial-setup-script/]

**Comment by stepmond** [comment]
[https://www.pcsoleil.fr/optimisation-de-windows-10-grace-a-win10-initial-setup-script/](url)

**Comment by entdark** [comment]
> After further consideration, I think a Spreadsheet might be a better format for storing this information.

Nice sheet. I don't understand it a bit: what every value mean in "Disassembler0 defaults", "Average Users", "Windows Default".
Because if 1 or 0 could be treated as enabled or disabled respectively then that whole understanding logic gets smashed against the wall when we see some magic 2 or 3 value.
Because there is not provided map in the sheet with EnabledFeature = 1, DisabledFeature = 0, EnabledAltFeature = 2 etc. (And seems like 2 is not an alt feature, just some undefined default value.)
You'd either have to create a map of every feature and number or just fill the values columns with EnableFeature options.

It gets even more uncertain for Install/Uninstall, Add/Remove etc options especially when there are Install/Uninstall order then Uninstall/Install just after that.

**Comment by ruehsn** [comment]
> what every value mean in "Disassembler0 defaults", "Average Users", "Windows Default".
see cell notes

> Because if 1 or 0 could be treated as enabled or disabled respectively then that whole understanding logic gets smashed against the wall when we see some magic 2 or 3 value.
0 = no change (neither enabling or disabling)
1-3 = runs the script option defined in columns K-M
I've add some additional doc on the 2nd tab

**Comment by klepper** [comment]
Someone went line by line of the last version of this script (Sept 11 2019) and explained what each line does.  I saw the post on Reddit. [Here is the link.](http://checkthebenchmarks.com/2019/09/11/win10-initial-setup-script/)
